tests/unit/test_project.py

```diff
@@ -106,50 +113,28 @@ def test_project_unmarshal(check, yaml_loaded_data):
         project.description,
         "this is an example of a rockcraft.yaml for the purpose of testing rockcraft",
     )
-    check.equal(project.rock_license, "Apache-2.0")
-    check.equal(project.build_base, "ubuntu:20.04")
-    check.equal(project.entrypoint, ["/bin/hello"])
-    check.equal(project.cmd, ["world"])
-    check.equal(project.env, [{"NAME": "VALUE"}])
-    check.equal(project.package_repositories, [{"type": "apt", "ppa": "ppa/ppa"}])
-    check.equal(
-        project.parts,
-        {
-            "foo": {
-                "plugin": "nil",
-                "overlay-script": "ls",
-            }
-        },
-    )
-
-
-def test_unmarshal_no_repositories(yaml_loaded_data):
-    yaml_loaded_data["package-repositories"] = None
-
-    project = Project.unmarshal(yaml_loaded_data)
-
-    assert project.package_repositories == []
-
-
-def test_unmarshal_undefined_repositories(yaml_loaded_data):
-    del yaml_loaded_data["package-repositories"]
-
-    project = Project.unmarshal(yaml_loaded_data)
-
-    assert project.package_repositories is None
-
+    assert project.rock_license == "Apache-2.0"
+    assert project.build_base == "ubuntu:20.04"
+    assert project.parts == {
+        "foo": {
+            "plugin": "nil",
+            "overlay-script": "ls",
+        }
+    }
 
-def test_unmarshal_invalid_repositories(yaml_loaded_data):
-    yaml_loaded_data["package-repositories"] = [{}]
 
-    with pytest.raises(ProjectValidationError) as error:
-        Project.unmarshal(yaml_loaded_data)
+@pytest.mark.parametrize(
+    "deprecated_field",
```

Comment:
can we refer to those fields as "removed" instead of "deprecated"?

Reply:
yep